### ISSUE TITLE
Please add this to Vencord as well!

### ISSUE THREAD

**Issue by Ndauch2501** [programmer2514/BetterDiscord-CollapsibleUI#114]
@programmer2514 Please consider adding your plugin to Vencord please!!

**Comment by programmer2514** [comment]
This plugin depends on Zeres Plugin Library and BDAPI - does Vencord have an equivalent? I have never developed for it before.

**Comment by Ndauch2501** [comment]
> This plugin depends on Zeres Plugin Library and BDAPI - does Vencord have an equivalent? I have never developed for it before.

please consider, importing to Vencord as well, esp when theres no other snippets/plugins that side performing this as well as yours do :)

**Comment by cheatfreak47** [comment]
I'd like to second this request, Vencord generally seems to perform much better then the other modified clients I've tried.

**Comment by O4bit** [comment]
I'd like to third this request, The guy above is right and i think discord is taking up to much space in my monitor

**Comment by O4bit** [comment]
> This plugin depends on Zeres Plugin Library and BDAPI - does Vencord have an equivalent? I have never developed for it before.

I haven't looked though the docs of vencord yet but, i think there will surely be some equivalent too them since there has been attempts to fork your plugin and remake it for vencord (https://github.com/Vendicated/Vencord/pull/2193)

**Comment by programmer2514** [comment]
Unfortunately, as I do not use Vencord, I am not the ideal candidate for porting this plugin to work in it. I am currently working on a rewrite that should make it much easier to port than it currently would be, but even then it would need to be significantly re-written and maintained separately from the base branch of CollapsibleUI.

If a Vencord developer would like to pick up the task of porting this project, I would be happy to work with them if needed, but I do not have the time or resources to learn an entirely new API, re-write this plugin in TSX, and maintain bugs and breakages in two separate, independent clients.

TLDR; This is a lot easier said than done, and is not currently on my radar for the forseeable future.
